### PR TITLE
Add API key endpoints methods

### PR DESCRIPTION
**Pull request by noclat**
- Add API key endpoints methods
- Update types
- Update docs
- Bump minor version

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -478,6 +483,76 @@ pk.patch.deleteLang('<patch-id>', 'fr');
...
+
+```js
+pk.keys.create('<role>', { domains: [] });
+```
+
+| Parameter | Type | Description |
+| --- | --- | --- |
+| `role` | `('public' \| 'private')` | API key role. |
+| `options` | `object?` | API key options. |
+| `options.domains` | `string[]?` | Domain or IP restriction (for public keys only). |
```
Comment (raphi) [inline README.md:510]: It actually works for both types of keys 👍

File: README.md [hunk 2]
```diff
@@ -478,6 +483,76 @@ pk.patch.deleteLang('<patch-id>', 'fr');
...
+
+| Parameter | Type | Description |
+| --- | --- | --- |
+| `id` | `string` | API key ID. |
+
+### `pk.keys.update()`
+
+⚠️ Restricted to **private** API keys, **do NOT expose the private key to the browser**.
+
+Update an API key.
```
Comment (raphi) [inline README.md:530]: `Update the domains list of the API key`

**Review by raphi — APPROVED** [review]
LGTM 👍

File: README.md (hunk 2 above)
Comment (noclat) [reply README.md:530]: Yeah for now, but left the `create` and `update` methods open to more options. I wouldn't want to have breaking changes the fay we add an option other than `domains`, we never know.

**Pull request merged**